feat(desktop-lite): make fluxbox optional (#1058)
devcontainers/features: src/desktop-lite/NOTES.md

See the [Fluxbox menu documentation](http://www.fluxbox.org/help/man-fluxbox-menu.php) for format details. More information on additional customization can be found in Fluxbox's [help](http://www.fluxbox.org/help/) and [general](http://fluxbox.sourceforge.net/docbook/en/html/book1.html) documentation.

> If you do not want to use Fluxbox, then you can set `installFluxbox` to `false` and install your own window manager.

Reviewer: ```suggestion
If you prefer not to use Fluxbox, you can set the `installFluxbox` Feature option to `false` and install your own window manager.
```
